app/controllers/api/v1/user_availabilities_controller.rb

```diff
@@ -1,15 +1,15 @@
 class Api::V1::UserAvailabilitiesController < Api::V1::BaseController
   before_action :activate_user
 
-  def create
-    @user_availability = UserAvailability.new(availability_params)
-    @user_availability.user = current_user
-    if @user_availability.save
-      render json: @user_availability
-    else
-      render_error
-    end
-  end
+  # def create
+  #   @user_availability = UserAvailability.new(availability_params)
+  #   @user_availability.user = current_user
+  #   if @user_availability.save
+  #     render json: @user_availability
+  #   else
+  #     render_error
+  #   end
+  # end
```

Comment:
if we are not going to use it anymore better delete the code and the route for readability sake